user_docs/en/changes.md

```diff
@@ -1,5 +1,14 @@
 # What's New in NVDA
 
+## 2024.4.1
+
+This patch release includes a fix for Outlook on certain Windows 10 systems.
```

Comment:
This should not be mentioned until we actually have a fix.

Reply:
```suggestion
```